packages/paste-core/components/time-picker/src/TimePicker.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {InputProps} from '@twilio-paste/input';
+import {Input} from '@twilio-paste/input';
+
+export type TimePickerProps = Omit<InputProps, 'type' | 'insertBefore' | 'insertAfter' | 'step'>;
+
+// @ts-ignore to remove step from props so it is not used
```

Comment:
What is the reported TS error here? I'm wondering if we can find a resolution rather than using `ts-ignore`.

Reply:
I added a better comment to try to address this! Also changed it to a `@ts-expect-error`. It's because we're omitting `step` from the type but still destructuring it from the time picker props.